sunpy/coordinates/transformations.py

```diff
@@ -187,71 +221,59 @@ def hcc_to_hgs(helioccoord, heliogframe):
                            "heliographic coordinates for observer '{}' "
                            "without `obstime` being specified.".format(helioccoord.observer))
 
-    x = helioccoord.x.to(u.m)
-    y = helioccoord.y.to(u.m)
-    z = helioccoord.z.to(u.m)
-
-    l0_rad = helioccoord.observer.lon
-    b0_deg = helioccoord.observer.lat
+    # Make sure the observer is in HGS
+    obs = helioccoord.observer.\
```

Comment:
I really hate line continuation characters, can't you line break after the `transform_to(`?

Reply:
Moot (since HGS is guaranteed)